org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/launch/GradleRunConfigurationAttributes.java

```diff
@@ -258,88 +290,83 @@ public static void applyShowConsoleView(boolean showConsoleView, ILaunchConfigur
         launchConfiguration.setAttribute(SHOW_CONSOLE_VIEW, showConsoleView);
     }
 
+    public static void applyWorkspaceOverride(boolean overrideWorkspaceSettings, boolean offlineMode, boolean buildScansEnabled, ILaunchConfigurationWorkingCopy launchConfiguration) {
```

Comment:
Because how the API on the Eclipse run configuration UI is structured. Each page can only update part of the complete run config. So for instance `GradleDistributionTab` calls `applyGradleDistribution`, `ArgumentsTab` calls `applyArgumentExpressions` and so forth.